src/test/java/org/kainos/ea/service/JobServiceUnitTest.java

```diff
@@ -59,9 +60,37 @@ void getAllJobs_shouldThrowFailedToGetJobsException_whenDaoThrowsSQLException()
     @Test
     void getAllJobs_shouldThrowFailedToGetJobsException_whenDatabaseConnectorThrowsSQLException() throws SQLException {
         Mockito.when(databaseConnector.getConnection()).thenThrow(SQLException.class);
+        Mockito.when(jobDao.getJobById(connection,-1)).thenReturn(null);
 
         assertThrows(FailedToGetJobsException.class,
                 () -> jobService.getAllJobs());
     }
 
+    @Test
+    void deleteJob_shouldReturnFalse_IfJobIdNotFound() throws SQLException, FailedtoDeleteException {
+        Mockito.when(databaseConnector.getConnection()).thenReturn(connection);
+        Mockito.when(jobDao.getJobById(connection,-1)).thenReturn(null);
+        Mockito.doNothing().when(jobDao).deleteJob(isA(Connection.class),isA(Integer.class));
```

Comment:
you don't need to do this, it'll do nothing by default

Reply:
Method was changed to throw not found exception, test changed to reflect this